modules/display_books.js

```diff
@@ -0,0 +1,58 @@
+class Book {
+  constructor() {
+    this.booksParent = document.getElementById('table-row');
+    this.titleInput = document.getElementById('title');
+    this.authorInput = document.getElementById('author');
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+    this.addBook = this.addBook.bind(this);
+    this.deleteBook = this.deleteBook.bind(this);
+    this.displayBooks();
+  }
+
+  addBook(e) {
+    e.preventDefault();
+    const title = this.titleInput.value;
+    const author = this.authorInput.value;
+    if (title && author) {
+      const newBook = { title, author };
+      this.books.push(newBook);
+      localStorage.setItem('books', JSON.stringify(this.books));
+      this.titleInput.value = '';
+      this.authorInput.value = '';
+      this.bookElement(newBook, this.books.length - 1);
+    }
+  }
+
+  bookElement(book, index) {
+    const bookContainer = document.createElement('td');
+    if (index % 2 === 0) {
+      bookContainer.classList.add('dark');
+    }
```

Comment:
- Great job for implementing using class, I think in this milestone you have to use an arrow function to fit the [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) 👍 
<img width="524" alt="Screenshot 2023-06-05 154539" src="https://github.com/AnsarIbrahim/Awesome-books-with-ES6/assets/52010789/d7ae5a67-d7ea-41f7-a937-071869c1036b">